### ISSUE TITLE
WHM Support

### ISSUE THREAD

**Issue by kenetik** [steadramon/cpanel_addon-twostepauth#6]
Not really an issue, just feedback. Love the project, would love to see integrated for WHM login as well rather than just cPanel accounts.

**Comment by steadramon** [comment]
Hey! Thanks for the feedback!

I'll be looking at WHM logins as part of the next release sprint. 

With cPanel also looking into this themselves they will be able to have better integration, I'm watching their progress, as they as most likely with this too.

**Comment by kenetik** [comment]
I have a very strong feeling that WHM/cPanel is monitoring the activity on this project extremely closely. They've received a lot of feedback for implementing two-factor auth. Disappointing it's taken them this long to show interest. Let me know if I can do anything to help. =)

**Comment by SkinnyBill** [comment]
Yeah, i did wonder myself. It seemed odd to protect cPanel accounts but leave the WHM account unprotected. Glad to see you're working on it.

kenetik - I agree. It's literally available everywhere now and they're still not doing anything about it. WHMCS claims to support it, but i don't know if that protects everything or just the billing information.

**Comment by steadramon** [comment]
Lack of WHM support was so I could gauge the reception of the plugin whilst releasing something of use :)

WHMCS isn't made by cPanel, so that just protects WHMCS and associate parts

**Comment by SkinnyBill** [comment]
Oh and also, regarding my little comment about "multiple accounts on the same server", ignore that.
I didn't notice that the app does indeed show the username, like so

my.cpanel.server
123 456
cPanel Username

But anyway, it appears iPhones don't like spaces, encoded or not, in the first description but doesn't mind them in the second. Looks like this is fixed.

**Comment by SkinnyBill** [comment]
Whoops, that was supposed to be on issue #5 :)

**Comment by kenetik** [comment]
@steadramon Feel free to close this if you'd like, not an issue, merely a suggestion.

**Comment by martymcguire** [comment]
+1 for WHM support. I feel that WHM is more critical to protect with 2FA than CPanel given that WHM is used to manage CPanel.

**Comment by madsmao** [comment]
+1
